src/components/ViewPage/ViewPage.tsx

```diff
@@ -0,0 +1,161 @@
+import * as React from 'react'
+import { Props, State } from './ViewPage.types'
+import { Page, Loader } from 'decentraland-ui'
+// import { t } from 'decentraland-dapps/dist/modules/translation/utils'
+import NotFoundPage from 'components/NotFoundPage'
+import Footer from 'components/Footer'
+import Navbar from 'components/Navbar'
+import { Project } from 'modules/project/types'
+
+import './ViewPage.css'
+import ViewMenu from './ViewMenu'
+import { t } from 'decentraland-dapps/dist/modules/translation/utils'
+import Icon from 'components/Icon'
+import { ComponentType } from 'modules/scene/types'
+import ViewPort from 'components/ViewPort'
+import ShortcutTooltip from 'components/ShortcutTooltip'
+import { Shortcut } from 'modules/keyboard/types'
+import Chip from 'components/Chip'
+
+export default class ViewPage extends React.PureComponent<Props, State> {
+  componentDidMount() {
+    const { currentProject, match, onLoadProject } = this.props
+    if (!currentProject && match.params.projectId) {
+      onLoadProject(match.params.projectId, match.params.type || 'public')
+    }
+  }
+
+  handlePreview = () => {
+    this.props.onPreview()
+  }
+
+  getParcelCount() {
+    const { currentProject } = this.props
+    if (!currentProject) {
+      return 0
+    }
+
+    return currentProject.layout.cols * currentProject.layout.rows
+  }
+
+  getObjectCount() {
+    const { currentScene } = this.props
+    if (!currentScene) {
+      return 0
+    }
+
+    const parcelCount = this.getParcelCount()
+    if (parcelCount === 0) {
+      return 0
+    }
+
+    let componentCount = 0
+    for (let component of Object.values(currentScene.components)) {
+      switch (component.type) {
+        case ComponentType.GLTFShape:
+        case ComponentType.NFTShape:
+          ++componentCount
```

Comment:
```suggestion
          componentCount++
```

Just to follow the convention used elsewhere in the code 🙏